roles/vdm/tasks/connect.yaml

```diff
@@ -20,13 +20,6 @@
   - name: connect - SAS/CONNECT FQDN to the SAN DNS list for cert-manager
     set_fact:
       V4_CFG_TLS_ADDITIONAL_SAN_DNS: "{{ V4_CFG_TLS_ADDITIONAL_SAN_DNS }} {{ V4_CFG_CONNECT_FQDN }}"
```

Comment:
Is the block still needed? Also, is the content needed here for the SAS/CONNECT FQDN? Just wondering if we don't have the transformer does this code help if the customer add this on their own.

Reply:
I can check into that further, I had initially removed it but then restored it to preserve the behavior since it was contributing to the manifest creation failure. Suspect we will want to update the CONFIG-VARS.md documentation surrounding that variable if we end up removing that block.

Reply:
@thpang, I removed the block per our discussion on Weds morning and restored the condition to what it was before the block that I removed was added.

Reply:
Resolved the merge conflicts by rebasing these changes to origin/staging just now